### DISCUSSION TITLE
Artist profile

### DISCUSSION THREAD

**Discussion by maH-muduL** [gokadzev/Musify#159] [Ideas]
Can we have artist profile option? 

This way we can enter someones profile and listen to their songs. It will be helpful.

Btw, thanks for such a minimalistic app. It works so smoothly.

**Comment by gokadzev** [comment]
> Can we have artist profile option?
> 
> This way we can enter someones profile and listen to their songs. It will be helpful.
> 
> Btw, thanks for such a minimalistic app. It works so smoothly.

Unfortunately I'm still not able to do this, the API of YouTube which i currently use doesn't support things like that and neither i know any better alternative for it. (only YouTube Music but it's not supported for many countries)
